test_cli_args.py

```diff
@@ -0,0 +1,91 @@
+import responses
+from unittest.mock import patch, MagicMock
+
+from doctoshotgun import Application, DoctolibDE, DoctolibFR, MasterPatientPage
+
+CENTERS = [
+    {
+        "name_with_title": "Doktor",
+        "city": "koln",
+    },
+    {
+        "name_with_title": "Doktor2",
+        "city": "koln",
+    },
+    {
+        "name_with_title": "Doktor",
+        "city": "neuss",
+    },
+]
+
+
+@responses.activate
+@patch('doctoshotgun.DoctolibDE')
+def test_center_arg_should_filter_centers(MockDoctolibDE, tmp_path):
+    """
+    Check that booking is performed in correct city
+    """
+    # prepare
+    mock_doctolib_de = get_mocked_doctolib(MockDoctolibDE)
+
+    # call
+    center = 'Doktor'
+    city = 'koln'
+    call_application(city, cli_args=['--center', center])
+
+    # assert
+    assert mock_doctolib_de.get_patients.called
+    assert mock_doctolib_de.try_to_book.called
+    for call_args_list in mock_doctolib_de.try_to_book.call_args_list:
+        assert call_args_list.args[0]['name_with_title'] == center
+        assert call_args_list.args[0]['city'] == city
+
+
+@responses.activate
+@patch('doctoshotgun.DoctolibDE')
+def test_center_exclude_arg_should_filter_excluded_centers(MockDoctolibDE, tmp_path):
+    """
+    Check that booking is performed in correct city
+    """
+    # prepare
+    mock_doctolib_de = get_mocked_doctolib(MockDoctolibDE)
+
+    # call
+    excluded_center = 'Doktor'
+    city = 'koln'
+    call_application(city, cli_args=['--center-exclude', excluded_center])
+
+    # assert
+    assert mock_doctolib_de.get_patients.called
+    assert mock_doctolib_de.try_to_book.called
+    for call_args_list in mock_doctolib_de.try_to_book.call_args_list:
+        assert call_args_list.args[0]['name_with_title'] != excluded_center
+        assert call_args_list.args[0]['city'] == city
+
+
+def get_mocked_doctolib(MockDoctolibDE):
+    mock_doctolib_de = MagicMock(wraps=DoctolibDE)
+    MockDoctolibDE.return_value = mock_doctolib_de
+
+    mock_doctolib_de.vaccine_motives = DoctolibDE.vaccine_motives
```

Comment:
Does someone with more python testing knowledge understand why this is required and not automatically set by the `MagicMock(wraps=DoctolibDE)` command?